From 5972cd58bde3bc8bacfe994e5b127c411241f255 Mon Sep 17 00:00:00 2001
From: law <law@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Tue, 3 Jan 2017 05:36:40 +0000
Subject: [PATCH] 	* config/darwin-driver.c (darwin_driver_init):
 Const-correctness 	fixes for first_period and second_period variables.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/trunk@244010 138bc75d-0d04-0410-961f-82ee72b054a4
---
diff --git a/gcc/config/darwin-driver.c b/gcc/config/darwin-driver.c
index 0c4f0cd..e3ed79d 100644
--- a/gcc/config/darwin-driver.c
+++ b/gcc/config/darwin-driver.c
@@ -299,10 +299,10 @@ darwin_driver_init (unsigned int *decoded_options_count,
   if (vers_string != NULL)
     {
       char *asm_major = NULL;
-      char *first_period = strchr(vers_string, '.');
+      const char *first_period = strchr(vers_string, '.');
       if (first_period != NULL)
 	{
-	  char *second_period = strchr(first_period+1, '.');
+	  const char *second_period = strchr(first_period+1, '.');
 	  if (second_period  != NULL)
 	    asm_major = xstrndup (vers_string, second_period-vers_string);
 	  else
